corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -52,7 +53,7 @@ def _build_case_relationships(self):
 
     def _add_module_contributions(self):
         for module in self.app.get_modules():
-            if isinstance(module, (Module, AdvancedModule)):
+            if isinstance(module, (Module, AdvancedModule, ShadowModule)):
```

Comment:
If I understand this correctly, this is now only excluding `ReportModule`, right? Would it be worth just removing this if statement all together? @orangejenny

Reply:
I just tried it out and it errors on `AttributeError: 'ReportModule' object has no attribute 'case_details'`

So this could check `hasattr(module, 'case_details')` instead of the specific classes, which would be more functionality-driven.

Reply:
I wondered about this. I swapped it over to the attribute check.